app/config/collections.php

```diff
@@ -1436,6 +1436,87 @@
             ],
         ],
     ],
+    Database::SYSTEM_COLLECTION_BUCKETS => [
+        '$collection' => Database::SYSTEM_COLLECTION_COLLECTIONS,
+        '$id' => Database::SYSTEM_COLLECTION_BUCKETS,
+        '$permissions' => ['read' => ['*']],
+        'name' => 'File',
+        'structure' => true,
+        'rules' => [
+            [
+                '$collection' => Database::SYSTEM_COLLECTION_RULES,
+                'label' => 'Date Created',
+                'key' => 'dateCreated',
+                'type' => Database::SYSTEM_VAR_TYPE_NUMERIC,
+                'default' => 0,
+                'required' => false,
+                'array' => false,
+            ],
+            [
+                '$collection' => Database::SYSTEM_COLLECTION_RULES,
+                'label' => 'Enabled',
+                'key' => 'enabled',
+                'type' => Database::SYSTEM_VAR_TYPE_BOOLEAN,
+                'default' => true,
+                'required' => false,
+                'array' => false,
+            ],
+            [
+                '$collection' => Database::SYSTEM_COLLECTION_RULES,
+                'label' => 'Name',
+                'key' => 'name',
+                'type' => Database::SYSTEM_VAR_TYPE_TEXT,
+                'default' => '',
+                'required' => true,
+                'array' => false,
+            ],
+            [
+                '$collection' => Database::SYSTEM_COLLECTION_RULES,
+                'label' => 'Adapter',
+                'key' => 'adapter',
+                'type' => Database::SYSTEM_VAR_TYPE_TEXT,
+                'default' => '',
+                'required' => true,
+                'array' => false,
```

Comment:
We should probably also add an attribute called `adapterCredentials` for future usage. This attribute can use the DB built in JSON filter for easier internal usage.

Reply:
added